lib/services/presets.go

```diff
@@ -494,42 +494,19 @@ func bootstrapRoleMetadataLabels() map[string]map[string]string {
 	}
 }
 
+var defaultAllowRulesMap = map[string][]types.Rule{
+	teleport.PresetAuditorRoleName: NewPresetAuditorRole().GetRules(types.Allow),
+	teleport.PresetEditorRoleName:  NewPresetEditorRole().GetRules(types.Allow),
```

Comment:
TBH I'm not sure why we have roles defined in two places, but the preset here https://github.com/gravitational/teleport/blob/9d2c2cbc1fbc408b9b92c29af4246e7b1c4d0043/lib/services/presets.go#L127-L165 looks different than the one you want to remove. The comment at the end of this function should be probably also updated if we make any changes.

Reply:
It looks different because it's meant to be, AFAICT, additive compared to the initial definition of the editor role. So this is supposed to be the delta between `editor` preset introduction to now.